app/components/mandatarissen/mandataris-edit-prompt.hbs

```diff
@@ -61,3 +82,31 @@
     />
   </div>
 </AuModal>
+<AuModal
+  @title={{(concat "Bekrachtig " @mandataris.bekleedt.bestuursfunctie.label)}}
+  @modalOpen={{this.isBekrachtiging}}
+  @closeModal={{this.cancel}}
+  as |Modal|
+>
+  <Modal.Body>
+    <p>
+      Bent u zeker dat u het mandaat wil bekrachtigen? Deze actie is niet
+      omkeerbaar.
+    </p>
+  </Modal.Body>
+  <Modal.Footer>
+    <AuToolbar as |Group|>
+      <Group>
+        <AuButtonGroup>
+          <AuButton {{on "click" this.onBekrachtig}}>
+            Bekrachtig
+          </AuButton>
+          <AuButton
+            {{on "click" this.cancel}}
+            @skin="secondary"
+          >Annuleer</AuButton>
+        </AuButtonGroup>
+      </Group>
+    </AuToolbar>
+  </Modal.Footer>
+</AuModal>
```

Comment:
Not sure if I should put the Modal in a component

Reply:
would be a good idea, but I can live with it.